README.rst

```diff
@@ -708,16 +770,67 @@ OpenVswitch Bridges
             use_interfaces:
             - eth1
 
-Linux with proxy
+
+Linux /etc/environment:
+``/etc/environment`` is for static system wide variable assignment after boot. Variable expansion is frequently not supported.
 
 .. code-block:: yaml
 
     linux:
-      network:
+      system:
+        env:
+          BOB_VARIABLE: Alice
+          ...
+          BOB_PATH:
+            - /srv/alice/bin
+            - /srv/bob/bin
+          ...
+          global_locales: |
+            LC_ALL=en_US
```

Comment:
I think this way should not be allowed for environment - you can only set key=value so stick with yaml.